apps/emqx_bridge_azure_event_hub/test/emqx_bridge_azure_event_hub_producer_SUITE.erl

```diff
@@ -11,7 +11,7 @@
 -include_lib("snabbkaffe/include/snabbkaffe.hrl").
 
 -define(BRIDGE_TYPE, azure_event_hub_producer).
--define(BRIDGE_TYPE_BIN, <<"azure_event_hub_producer">>).
```

Comment:
I think this shouldn't be altered.  The bridge is still AEH producer, not Kafka.  :thinking:

Reply:
My point was that we don't need to change this and affect all testcases, only `t_same_name_azure_kafka_bridges` needs this special bridge type.

Reply:
Got it